Microsoft.Toolkit.Uwp.UI.Controls/Expander/Expander.xaml

```diff
@@ -0,0 +1,313 @@
+                        <VisualStateManager.VisualStateGroups>
+                            <VisualStateGroup x:Name="CommonStates">
+                                <VisualState x:Name="Normal" />
+                                <VisualState x:Name="ReadOnly" />
+                                <VisualState x:Name="MouseOver" />
+                                <VisualState x:Name="Disabled" />
+                                <VisualState x:Name="Enabled" />
+                            </VisualStateGroup>
+                            <VisualStateGroup x:Name="HeaderStates">
+                                <VisualState x:Name="HeaderExpanderMouseOver" />
+                                <VisualState x:Name="HeaderSelectorMouseOver" />
+                                <VisualState x:Name="HeaderExpanderPressed" />
+                                <VisualState x:Name="HeaderSelectorPressed" />
+                            </VisualStateGroup>
+                            <VisualStateGroup x:Name="ContentStates">
+                                <VisualState x:Name="ContentExpanded">
+                                    <VisualState.Setters>
+                                        <Setter Target="PART_MainContentRow.Height" Value="Auto" />
+                                    </VisualState.Setters>
+                                </VisualState>
+                                <VisualState x:Name="ContentCollapsed">
+                                    <VisualState.Setters>
+                                        <Setter Target="PART_MainContentRow.Height" Value="0" />
+                                    </VisualState.Setters>
+                                </VisualState>
+                            </VisualStateGroup>
+                        </VisualStateManager.VisualStateGroups>
+                        <Grid Grid.RowSpan="2" VerticalAlignment="Top">
+                            <Grid.RowDefinitions>
+                                <RowDefinition Height="Auto" />
+                                <RowDefinition Height="Auto" x:Name="PART_MainContentRow" />
+                            </Grid.RowDefinitions>
+                            <Grid.ColumnDefinitions>
+                                <ColumnDefinition Width="Auto" />
+                                <ColumnDefinition />
+                            </Grid.ColumnDefinitions>
+                            <ToggleButton x:Name="PART_ExpanderToggleButton" Height="40"
+                                          Style="{StaticResource HeaderToggleButtonStyle}" VerticalAlignment="Top"
+                                          IsChecked="{Binding IsExpanded, Mode=TwoWay, RelativeSource={RelativeSource TemplatedParent}}"
+                                          Width="40">
+                                <SymbolIcon Symbol="Forward" />
+                            </ToggleButton>
+                            <Grid x:Name="PART_Header" VerticalAlignment="Stretch"
```

Comment:
Put the header into the toggle button. 
```
<ToggleButton ContentTemplate="{TemplateBinding HeaderTemplate}" />
```

Then in the ToggleButton style that you defined, put the arrow:
```
<Grid>
    <Arrow Grid.Column="0"/>
    <ContentPresenter="1"/>
</Grid>
```